docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -236,6 +236,54 @@ If a Jupyter kernel with the name `kedro_<package_name>` already exists then it
 
 You can use the `jupyter kernelspec` set of commands to manage your Jupyter kernels. For example, to remove a kernel, run `jupyter kernelspec remove <kernel_name>`.
 
+### Debugging with %debug and %pdb
+
+ `%debug` will run post-mortem debugging using IPython. Running `%debug` immediately after an error has occurred will:
```

Comment:
Additionally, `debug` can be used not just for post-mortem, but to run a line or cell in debug mode from the get-go. I don't think it would be a miss to include those details as well - I had a brief description in [this commit](https://github.com/kedro-org/kedro/pull/3579/commits/b7ad523b0c72461cdd40e7a85d05b818028e5249#diff-fc0375e3319c0a3fc553add23334fff3fd5dc8df14d02be33d88ad04b6efd6a4); I'm not tied to the wording or structuring, but I do think the content should feature somewhere in this section.